src/main/java/org/pantsbuild/jarjar/Wildcard.java

```diff
@@ -122,15 +122,12 @@ private static boolean checkIdentifierChars(String expr, String extra) {
       if (expr.endsWith("package-info")) {
           expr = expr.substring(0, expr.length() - "package-info".length());
       }
-      // Allow patterns under META-INF if you want to match multi-release class files.
-      if (expr.startsWith("META-INF/")) {
-          expr = expr.substring("META-INF/".length(), expr.length());
-      }
       for (int i = 0, len = expr.length(); i < len; i++) {
           char c = expr.charAt(i);
           if (extra.indexOf(c) >= 0)
               continue;
-          if (!Character.isJavaIdentifierPart(c))
+          // Dash ('-') support added to support with scala packages
```

Comment:
The wording here is off, how about:
// Dash ('-') support added to accommodate scala packages